pyeudiw/satosa/backend.py

```diff
@@ -56,8 +57,8 @@ def entity_configuration(self, context, *args):
         jwk = JWK()
 
         data = {
-            "exp": 1649590602,
-            "iat": 1649417862,
+            "exp": int((datetime.now() + timedelta(minutes=6)).timestamp()),
```

Comment:
minutes = 6

please use a general config time delta

TOKEN_EXP_DELTA or something like that

Reply:
Resolved in [14e8565](https://github.com/italia/eudi-wallet-it-python/pull/14/commits/14e8565f4c4a4faf0d274567db51407bc74f10d4)